srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -197,11 +216,13 @@ private void parseWeightResults(final Patient patient, final List<String> weight
         LOGGER.debug("Weight 6 months ago: {}", patient.getWeight6MonthsAgo());
     }
     
-    private void parseRecentVitalResults(final Patient patient, final List<String> vitalResults) throws ParseException
+    private void parseRecentVitalResults(final Patient patient, final List<String> vitalResults)
+            throws ParseException
     {
+        final SimpleDateFormat dateFormat = new SimpleDateFormat(
         // The date inside of returned vitals is inside of parentheses.
         // For example, pulse is returned as: "Pulse:       (03/05/10@09:00)  74  _NURSE,ONE_Vitals"
-        final SimpleDateFormat dateFormat = new SimpleDateFormat("(" + VISTA_DATE_OUTPUT_FORMAT.toPattern() + ")");
+                "(" + VISTA_DATE_OUTPUT_FORMAT.toPattern() + ")");
```

Comment:
Why did you separate the declaration and part of the constructor from this line?